openedx/edx-ora2
# fix: styles of tables in "Open Responses" section of instructor dashboard

**TL;DR -** Moved the ORA specific styling fixes from [platform code](https://github.com/openedx/edx-platform/commit/1dab744ba2785ec220cc40d7d8a932393b46f23f) to the ora code.

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**

- Adds padding to the cells of the tables that are rendered in the "Open Responses" tab in Instructor Dashboard of the LMS.

**Developer Checklist**


**Testing Instructions**

This should be reviewed in tandem with https://github.com/openedx/edx-platform/pull/35474

* Check out edx-platform to the PR branch from https://github.com/openedx/edx-platform/pull/35474
* Install this PR's branch in your instance/container and verify that the tables are cleaner.

Before:
![ora_before](https://github.com/user-attachments/assets/309242ce-25fe-4767-a0be-295b70ace120)

After:
![ora_after](https://github.com/user-attachments/assets/e6281ef3-dde6-4d0e-b423-32f2e0b12343)


**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
